Lib/test/support/os_helper.py

```diff
@@ -716,3 +717,37 @@ def __exit__(self, *ignore_exc):
             else:
                 self._environ[k] = v
         os.environ = self._environ
+
+
+try:
+    import ctypes
+    kernel32 = ctypes.WinDLL('kernel32', use_last_error=True)
+except (ImportError, AttributeError):
+    def subst_drive(path):
+        raise NotImplementedError('ctypes or kernel32 is not available')
+else:
+    ERROR_FILE_NOT_FOUND = 2
+    DDD_REMOVE_DEFINITION = 2
+    DDD_EXACT_MATCH_ON_REMOVE = 4
+    DDD_NO_BROADCAST_SYSTEM = 8
```

Comment:
I think these definitions should be moved to the try block after `kernel32` is defined. Then they're only defined when necessary.